loops.js

```diff
@@ -29,6 +36,15 @@
 // 3 is oneven
 // .... etc.
 // 15 is oneven
+for (i = 0; i<15 ; i++ ){
+    if (i % 2 === 0){
+        console.log(i, "is even");
+    } else {
+        console.log(i, "is oneven");
+    }
+}
+
+
 
```

Comment:
je loopje is eentje te kort, mag lopen tot i < 16 omdat je getal 15 ook wil. Ik zie in de terminal dat de uitkomst verder goed is. Begrijp ik hieruit dat de komma hetzelfde kan werken als een plusje en spatie? 

Er staat een paar schoonheidsfoutjes in. Er missen spaties en ; mag meteen na de 16.

Reply:
![Screenshot 2023-01-30 101657](https://user-images.githubusercontent.com/77756218/215436110-a1355266-e9c1-421a-9638-40e5041fe3d9.png)

Wist het verschil niet 100% zeker, maar dit is dus hoe het werkt, Maar met een nummer / variabele naam geeft hij het weer op 1 regel.

Reply:
Ik dacht ik ga het even testen in mijn code, maar het geeft gewoon een spatie weer ipv een nieuwe regel. Dus wat nou echt het verschil is durf ik niet te zeggen!

Reply:
Wat gek, want ik heb je huiswerk gecloned en gelogd in de terminal en dan komt er wel de zin keurig uit op een regel en niet op verschillende regels als je die , gebruikt.

Reply:
![2023-01-30](https://user-images.githubusercontent.com/107489497/215437666-ead8998f-2968-4820-94f5-3db162624739.png)

Reply:
Vreemd. De uitkomst met de verschillende operators is wel hetzelfde in de terminal. Ik ga er nog eens op googlen.

Reply:
https://stackoverflow.com/questions/33339688/why-comma-and-plus-log-the-console-output-in-different-pattern

Met de komma krijg je dus wel de spatie, maar niet wat je eigenlijk als uitkomst wil. Misschien geeft dit een duidelijk antwoord.